### DISCUSSION TITLE
Message sent to one alias is labeled with a different alias.

### DISCUSSION THREAD

**Discussion by nomonkeynodeal** [simple-login/app#1275] [Q&A]
I ordered a product a month ago and gave them my alias. I received the tracking number at that alias from pkginfo@ups.com.

I placed an order with a completely different company a few days ago, under a completely different alias, and again received the tracking number from pkginfo@ups.com. However, this time it was the same alias as the first order from a few months ago. It was not the alias I used to place the order.

To clarify, the recipient address in the email I received is an alias I never gave to the company I ordered from. I know this because I checked and they have a different email. This means that SL internally mixed up the aliases.

**Comment by nguyenkims** [comment]
SimpleLogin simply forwards all emails sent to your alias to your mailbox and doesn't know whether the alias shouldn't be used by the sender.

**Reply by nomonkeynodeal** [reply]
The recipient address in the email I received is an alias I never gave to the company I ordered from. I know this because I checked and they have a different email. This means that SL internally mixed up the aliases.

**Reply by nguyenkims** [reply]
Unfortunately we don't look at the email content and don't have any idea on what can be the cause of this issue.
